From 7a6d834c0104304f506852eddc25b22e1ce65e3b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=A9r=C3=A9my=20Rosen?= <jeremy.rosen@smile.fr>
Date: Fri, 26 Apr 2019 00:49:57 +0200
Subject: [PATCH 16/29] Document /etc/initrd-release

When systemd is started, it detects initrd by checking for that file
The usage of that file is not documented anywhere, so mention it early
in the most relevant man-page I could find.

(cherry picked from commit 6e9d2465923f8b3577cd7cfc7ef886808ded8122)
---
 man/bootup.xml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/man/bootup.xml b/man/bootup.xml
index 9468a61319..d371523400 100644
--- a/man/bootup.xml
+++ b/man/bootup.xml
@@ -156,7 +156,9 @@
     using systemd as well. In this case, boot up inside the initrd
     follows the following structure.</para>
 
-    <para>The default target in the initrd is
+    <para>systemd detects that it is run within an initrd by checking
+    for the file <filename>/etc/initrd-release</filename>.
+    The default target in the initrd is
     <filename>initrd.target</filename>. The bootup process begins
     identical to the system manager bootup (see above) until it
     reaches <filename>basic.target</filename>. From there, systemd
-- 
2.21.0

